Subject: [PATCH] drivers/input/serio/hp_sdc.c needs <linux/semaphore.h>

From: Geert Uytterhoeven <geert@linux-m68k.org>

drivers/input/serio/hp_sdc.c: In function 'hp_sdc_take':
drivers/input/serio/hp_sdc.c:198: error: implicit declaration of function 'up'

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 drivers/input/serio/hp_sdc.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/input/serio/hp_sdc.c
+++ b/drivers/input/serio/hp_sdc.c
@@ -69,6 +69,7 @@
 #include <linux/time.h>
 #include <linux/slab.h>
 #include <linux/hil.h>
+#include <linux/semaphore.h>
 #include <asm/io.h>
 #include <asm/system.h>
 
